bench/src/metrics.rs

```diff
@@ -35,6 +35,9 @@ impl AddAssign<&Self> for Metric {
 
 impl DivAssign<u64> for Metric {
     fn div_assign(&mut self, rhs: u64) {
+        if rhs == 0 {
+            return;
+        }
```

Comment:
where is this used? i think panic is more appropriate here than return

Reply:
avg metric. if there were no runs then bench exits with a log